go/integTest/glide_test_suite_test.go

```diff
@@ -55,8 +55,8 @@ func (suite *GlideTestSuite) SetupSuite() {
 		suite.T().Fatal(err.Error())
 	}
 
-	suite.redisVersion = extractServerVersion(string(byteOutput))
-	suite.T().Logf("Redis version = %s", suite.redisVersion)
+	suite.serverVersion = extractServerVersion(string(byteOutput))
```

Comment:
Please update `extractServerVersion` in a future PR according to #2167.

Reply:
Acknowledged.